ui/tests/acceptance/clients-list-test.js

```diff
@@ -44,16 +39,68 @@ module('Acceptance | clients list', function(hooks) {
 
     assert.equal(nodeRow.id, node.id.split('-')[0], 'ID');
     assert.equal(nodeRow.name, node.name, 'Name');
-    assert.equal(nodeRow.status, node.status, 'Status');
-    assert.equal(nodeRow.drain, node.drain + '', 'Draining');
-    assert.equal(nodeRow.eligibility, node.schedulingEligibility, 'Eligibility');
+    assert.equal(nodeRow.status.text, 'draining', 'Combined status, draining, and eligbility');
     assert.equal(nodeRow.address, node.httpAddr);
     assert.equal(nodeRow.datacenter, node.datacenter, 'Datacenter');
     assert.equal(nodeRow.allocations, allocations.length, '# Allocations');
   });
 
+  test('client status, draining, and eligibility are collapsed into one column', async function(assert) {
+    server.createList('agent', 1);
+
+    server.create('node', {
+      modifyIndex: 4,
+      status: 'ready',
+      schedulingEligibility: 'eligible',
+      drain: false,
+    });
+    server.create('node', {
+      modifyIndex: 3,
+      status: 'initializing',
+      schedulingEligibility: 'eligible',
+      drain: false,
+    });
+    server.create('node', {
+      modifyIndex: 2,
+      status: 'down',
+      schedulingEligibility: 'eligible',
+      drain: false,
+    });
+    server.create('node', {
+      modifyIndex: 1,
+      status: 'ready',
+      schedulingEligibility: 'ineligible',
+      drain: false,
+    });
+    server.create('node', {
+      modifyIndex: 0,
+      status: 'ready',
+      schedulingEligibility: 'ineligible',
+      drain: true,
+    });
```

Comment:
Nice fixtures. Setting `modifyIndex` to force the ordering you want is a neat trick. Only thing I'll say is that [there are traits for drain you can use here](https://github.com/hashicorp/nomad/blob/master/ui/mirage/factories/node.js#L36-L64).

Doesn't save you much, but at least we can control changes to the drain API in a more central place.

```js
server.create('node', 'draining', {
  modifyIndex: 0,
  status: 'ready'
});
```

Reply:
ah yes, I changed it accordingly, thanks. I tend to underuse traits.